default.nix

```
@@ -11,6 +11,7 @@ let
 
   pkgs = nixpkgs {
     overlays = [ (import ./overlay.nix) ];
+    system = "x86_64-linux";
     inherit (prev) config;
```

```suggestion
    inherit (prev) config system;
```
That should be cleaner?

---

Ah of course, even better!